doc/modules/ensemble.rst

```diff
@@ -128,16 +128,17 @@ Random Forests
 --------------
 
 In random forests (see :class:`RandomForestClassifier` and
-:class:`RandomForestRegressor` classes), each tree in the ensemble is
-built from a sample drawn with replacement (i.e., a bootstrap sample)
-from the training set. In addition, when splitting a node during the
-construction of the tree, the split that is chosen is no longer the
-best split among all features. Instead, the split that is picked is the
-best split among a random subset of the features. As a result of this
-randomness, the bias of the forest usually slightly increases (with
-respect to the bias of a single non-random tree) but, due to averaging,
-its variance also decreases, usually more than compensating for the
-increase in bias, hence yielding an overall better model.
+:class:`RandomForestRegressor` classes), each tree in the ensemble is built
+from a sample drawn with replacement (i.e., a bootstrap sample) from the
+training set. When splitting a node during the construction of the tree, the
+best split can be computed using either all input features
+(``max_features=’auto’``) or using the best feature subset of size
```

Comment:
It should indeed be clearer that in RFC, `max_features='auto'` indicates using the square root of the number of input features.

Reply:
Do you think it could also make sense to just have defaults 'sqrt' (for classifier) and None (for regressor) (or introduce a new string 'max' or 'all' or sth for max_features='features') and remove the whole 'auto' option? as 'auto' seems ambiguous anyways?

Reply:
+1 but this would require a deprecation cycle.